src/main/java/Task.java

```diff
@@ -0,0 +1,16 @@
+public abstract class Task {
+    protected Boolean done;
```

Comment:
Consider changing the name of the `done` variable to `isDone`, `hasDone` etc. This is according to the coding standard section "Boolean variables/methods should be named to sound like booleans" from https://se-education.org/guides/conventions/java/intermediate.html.